explorer/interpreter/impl_scope.h

```diff
@@ -42,6 +42,10 @@ class TypeChecker;
 // scope.
 class ImplScope {
  public:
+  ImplScope() {}
```

Comment:
Should this be explicit?

Reply:
```suggestion
  explicit ImplScope() {}
```
Sure, that seems reasonable.